### DISCUSSION TITLE
Badge width

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#2183] [Q&A] [answered]
Does the width of the badge has to hug the content? 
In my use case, it looks weird so was wondering if I can set the width to fit the longest word and use that width regardless of the content. 

![image](https://user-images.githubusercontent.com/64559618/152408689-ec02ddd5-089d-4865-8fe0-3346bc15557b.png)
![image](https://user-images.githubusercontent.com/64559618/152409152-b84b420d-b937-4b0d-b06a-749b99c0cdce.png)

Pic 1 is what the styling is now vs pic 2 is what I was considering. Thoughts?

**Comment by gloriliale** [comment, marked as answer]
Hi @pguptaDS! One of the main reasons we do want Badge to size itself around its contents is for internationalization. That way, if the content changes width bc of a language change, the component does the same. If there were a set width here, there's a higher chance of something not fitting, or even breaking in the same situation. 

The look of it right aligned looks good from our end, but if you could also try to center the badge in the cell, which might lean it closer to the look you were aiming for.

**Comment by pguptaDS** [comment]
Got it. Do you mean like this? 
![image](https://user-images.githubusercontent.com/64559618/152444279-96a71825-a443-4f01-8807-728ff088aedf.png)

**Reply by gloriliale** [reply]
Yup! We don't have guidelines on putting badge inside of table cells, but wanted to suggest centering since it resembled the style of the badges w/ a set width a bit more.
